sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -662,6 +662,23 @@ def _add_argparse_args(cls, parser):
                          'in the pipeline when running using the Fn API.'))
 
 
+class FlinkOptions(PipelineOptions):
+
+  @classmethod
+  def _add_argparse_args(cls, parser):
+    parser.add_argument('--flink_master',
+                        type=str,
+                        help=
+                        ('Addres of the Flink master where the Pipeline '
+                         'should be executed. Can either be of the form '
+                         '\'host:port\' or one of the special values '
+                         '[local], [collection], or [auto].'))
+    parser.add_argument('--parallelism',
```

Comment:
Should this be --flink_parallelism?

Reply:
Perhaps, flink-master should just be master? Or does this potentially clash with the global namespace?

Reply:
Flags are inherently a global namespace.

Reply:
The problem here is that we want to stay consistent with `FlinkPipelineOptions`, so I'd keep it as it is to not introduce a source of confusion.

Reply:
Ahh... yes, that's a valid reason.